### PR TITLE
Closes #3762: Re-save and re-export all views config in order to capture config drift

### LINKED ISSUE #3762: Re-save and re-export all views config in order to capture config drift.
At the moment, any minor change to an old view that is then exported for a pull request adds lots of config drift making it difficult to review pull requests that change view settings.

Basically, the way that Drupal exports views config now, is different from when we originally exported the views config in quickstart, and we need to capture those changes in config code.

This issue is usually the result of config schema changing upstream.

## Conditions of satisfaction
Each [existing view](https://github.com/search?q=repo%3Aaz-digital/az_quickstart%20path%3Aviews.view.&type=code) should be re-saved and re-exported in order to capture all of the updates to views config exports.

### PR DESCRIPTION
**Pull request by ejsamboy**
## Description
Closes: #3762

Re-saved and re-exported the existing Views configurations to resolve configuration drift issues when changes are made to old Views and then exported for a pull request.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change requires release notes.

### CODE REVIEW

File: modules/custom/az_person/config/install/views.view.az_person.yml [hunk 1]
```diff
@@ -107,25 +108,79 @@ display:
           items_per_page: 0
           total_pages: null
           id: 0
-          pagination_heading_level: h4
       exposed_form:
-        type: basic
+        type: az_better_exposed_filters
```
Comment (joeparsons) [inline modules/custom/az_person/config/install/views.view.az_person.yml:112]: Wondering if this is an expected change (?) 🤔

Comment (tadean) [reply modules/custom/az_person/config/install/views.view.az_person.yml:112]: I think this represents `az_finder` being enabled during export, maybe...

**Comment by danahertzberg** [comment]
I propose merging this first 😬 https://github.com/az-digital/az_quickstart/pull/3610

File: modules/custom/az_person/config/install/views.view.az_person.yml (hunk 1 above)
Comment (trackleft) [reply modules/custom/az_person/config/install/views.view.az_person.yml:112]: Correct, this view should be exported without the finder module enabled.

Comment (ejsamboy) [reply modules/custom/az_person/config/install/views.view.az_person.yml:112]: 👍

File: modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_row_with_background.yml
Comment (trackleft) [inline modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_row_with_background.yml:1]: This is not a view config file

File: modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_row.yml
Comment (trackleft) [inline modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_row.yml:1]: This is not a view config file

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_text_layout.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_text_layout.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_hero_layout.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_hero_layout.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_70_30_layout_image_right.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_70_30_layout_image_right.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_50_50_layout_image_right.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_50_50_layout_image_right.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_50_50_layout.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_50_50_layout.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_30_70_layout.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_news.az_marketing_cloud_30_70_layout.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_event.az_marketing_cloud_event_layout.yml
Comment (trackleft) [inline modules/custom/az_marketing_cloud/config/install/core.entity_view_display.node.az_event.az_marketing_cloud_event_layout.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_flexible_page/config/install/core.entity_view_display.node.az_flexible_page.az_row_with_background.yml
Comment (trackleft) [inline modules/custom/az_flexible_page/config/install/core.entity_view_display.node.az_flexible_page.az_row_with_background.yml:1]: This is not a view config file, remove this from the PR.

File: modules/custom/az_publication/config/install/views.view.az_publications.yml [hunk 2]
```diff
@@ -659,7 +659,7 @@ display:
           exposed: false
           draggable_views_reference: 'az_publications_curate:curation_page'
           draggable_views_null_order: before
-          draggable_views_pass_arguments: true
+          draggable_views_pass_arguments: 1
```
Comment (trackleft) [inline modules/custom/az_publication/config/install/views.view.az_publications.yml:662]: I made a new issue about this https://www.drupal.org/project/draggableviews/issues/3481312

**Comment by ejsamboy** [comment]
After having several problems with this PR, I created a new PR and branch to solve the views configuration drift issue.
https://github.com/az-digital/az_quickstart/pull/3876
newissue/3762

**Pull request closed without merging**